### DISCUSSION TITLE
Problem with export to excel

### DISCUSSION THREAD

**Discussion by pitrusek86** [DrCoffey/DeepSqueak#191] [General]
Hi,
I've got problem with export data to Excel. In the console, below error pops up:
![problem_with_export_to_excel](https://user-images.githubusercontent.com/122601301/227865269-31fc461c-da37-44e5-8ba9-0a68d675ef16.jpg)
Do you know how to fix it? Thanks in advance.

**Comment by DrCoffey** [comment]
Hey @pitrusek86, you may have a call at the very end of the file with a box that extends after the audio. It is a rare issue but that may be what is happening here.

Also, I just pushed a major update with a lot of bug fixes. Try downloading the new version, and make sure you are using a recent release of MATLAB (2021a or later). Let me know if that fixes your problem.
